flux/thermal.pyx

```diff
@@ -7,6 +7,11 @@ cdef extern from "pcc/thrmlLib.h":
 		     Qnp1, double T[], double ti[], double rhoc[],
 		     double emiss, double Fgeotherm, double *Fsurf)
 
+cdef extern from "pcc/thrmlLib.h":
```

Comment:
don't need this line---if you remove it, but keep the declaration of conductionT the same on the following lines, it will be in the same scope as the `cdef extern ...` started on the line above